### ISSUE TITLE
Two Resources at the same time

### ISSUE THREAD

**Issue by mcerezo2012** [maxpowel/jQuery-XMPP-plugin#28]
Hi there is it Possible to have 2 or more resource connection while having a conversation individually?.

Let's say for example there is a site which is connected to openfire server (using jquery-xmpp plugin). 

Let say 'user1' is connected to the openfire server and there's a list of his/her contacts there and when he/she clicked on 'user2' that is listed on his/her contact, a pop-up window will appear and another connection or resources will open, the same with other remaining contacts.
- I am using here an "iframe" tag in the pop-up box by passing a parameter and connecting it to the server.

Basically i am trying to achieved the chat feature of Google Plus Site, specifically the pop-up thing.(https://plus.google.com/)

Problem:
- My Openfire server crashes instantly when i am connected to the sever
- Theres a delayed message and sometimes the message does not delivered

Any suggestion how to achieved this?

I just needed it badly. :(

BTW i am using here Openfire 3.7.1 and i am using your plugin now for about 6months

Any reply would be so much appreciated. Thanks.

**Comment by maxpowel** [comment]
Hi! Glad to see the plugin is useful.

Try to use different resources. For example, in the main window use the resource "main" and in the popup use "popup".

The problem is openfire cant manage 2 or more connections at the same time using the same jid. By this way, you are using 2 different jids (user@domain.com/main and user@domain.com/popup).

**Comment by mayursnarole** [comment]
Hello,
I am having the same problem of using same resource, but I don't want to create too many resources on openfire as my clients count is much more.. Instead of that I want to attach the existing session, where I am using popup as resource. Is it possible to do this stuff. And if possible then please give solution with example..

Waiting for your response...
Thanks..
